django/contrib/auth/tests/urls_admin_cutom_pk.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
We need explicitly register new admin model, I've created separate file for this to have ability to 
```python
ROOT_URLCONF='django.contrib.auth.tests.urls_admin_cutom_pk'
```